app/serializers/event_serializer.rb

```diff
@@ -19,6 +18,12 @@ def address
     "#{object.address1} #{object.address2}"
   end
 
+  def exception_notes
+    exception_note = object.exception_notes(@instance_options[:zip_code])
+
+    exception_note || ''
```

Comment:
I started doing this in recent cases where data for the JSON fields is sporadic or only applicable in certain conditions, e.g., event_details, event and agency distance, and exception_note.  
It could be I just don't like seeing them.  Guarding against nulls could be useful in showing the data element type, for example an array versus a string.  Also, would like to know if there is a preference for the UI side.  Perhaps we can discuss in the standup and do a simple PR if we want to change it across the board?

Reply:
I'm cool with it either way. Sounds like you've put some thought into already.